packages/cashier/src/stores/payment-agent-store.ts

```diff
@@ -1,39 +1,54 @@
-import { action, computed, observable } from 'mobx';
+import { action, computed, observable, IObservableArray } from 'mobx';
+import { PaymentAgentDetailsResponse } from '@deriv/api-types';
 import { formatMoney, routes } from '@deriv/shared';
 import Constants from 'Constants/constants';
 import ErrorStore from './error-store';
 import VerificationStore from './verification-store';
+import {
+    TAgent,
+    TExtendedPaymentAgentList,
+    TExtendedPaymentAgentListResponse,
+    TPartialPaymentAgentList,
+    TPaymentAgentWithdrawConfirm,
+    TPaymentAgentWithdrawRequest,
+    TPaymentAgentWithdrawReceipt,
+    TRootStore,
+    TSupportedBank,
+    TTarget,
+    TWebSocket,
+} from 'Types';
 
 export default class PaymentAgentStore {
-    constructor({ WS, root_store }) {
+    constructor(public WS: TWebSocket, public root_store: TRootStore) {
```

Comment:
@george-usynin-binary What is the use of access modifiers here?

Reply:
@Bahar-FS , this is a short syntax in Typescript, and `public` access modifier because by default, all members (properties, fields, methods, or functions) of classes are public accessible from outside the class.

As mentioned this is a short syntax, please refer the image below:
<img width="1559" alt="Screenshot 2022-09-09 at 10 17 20" src="https://user-images.githubusercontent.com/103181646/189293517-aedfd27e-d3d7-476d-855d-5d9c3cee5a25.png">